Python/tigre/algorithms/pocs_algorithms.py

```diff
@@ -128,6 +130,10 @@ def __init__(self, proj, geo, angles, niter, **kwargs):
     def run_main_iter(self):
         stop_criteria = False
         n_iter = 0
+        if (self.regularization=="PICCS"):
+            # Modify the `set_res` function in the `Python\tigre\algorithms\iterative_recon_alg.py` file as needed.
+            res_prior = copy.deepcopy(self.res)

```

Comment:
hum not sure about this. `res` is the current variable right? You may want to initialize the algorithm to one image, but have a prior of another, so I think there should be a new kwarg for `prior`